src/utils/communitySbt/getSubgraphRoot.ts

```diff
@@ -0,0 +1,47 @@
+import { ApolloClient, InMemoryCache, gql, HttpLink } from '@apollo/client';
+import fetch from 'cross-fetch';
+import { Bytes } from 'ethers';
+
+const tokensQuery = `
```

Comment:
isn't a better variable name "roots query" unless missing smth?

Reply:
true, fixed in c6cd67a37d9d9fec45c474301bff74e2b8f7b205